src/XML/src/FMI/fmi_xml_terminals_and_icons_scheme.h

```diff
@@ -0,0 +1,62 @@
+/*
+    Copyright (C) 2023 Modelon AB
```

Comment:
```suggestion
    Copyright (C) 2024 Modelon AB
```
again feel free to ignore.